http-server-netty/src/test/groovy/io/micronaut/http/server/netty/filters/FilterBodySpec.groovy

```diff
@@ -0,0 +1,61 @@
+package io.micronaut.http.server.netty.filters
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.annotation.Requires
+import io.micronaut.core.annotation.Order
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.MediaType
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.annotation.RequestFilter
+import io.micronaut.http.annotation.ServerFilter
+import io.micronaut.http.client.HttpClient
+import io.micronaut.runtime.server.EmbeddedServer
+import jakarta.inject.Singleton
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Sinks
+import spock.lang.Specification
+
+class FilterBodySpec extends Specification {
+    def "delayed body"() {
```

Comment:
can you add another test that tests the case where a controller method is reached after a request filter processes the delayed body

Reply:
ok